src/pages/patientView/clinicalInformation/getClinicalInformationData.ts

```diff
@@ -21,13 +21,13 @@ function transformClinicalInformationToStoreShape(patientId: string, studyId: st
         clinicalData: clinicalDataPatient
     };
 
-    const samples: Array<ClinicalDataBySampleId> = chain(clinicalDataSample)
-        .groupBy('id')
-        .map((v, k) => ({
-            clinicalData: v,
-            id: k + '',
-        }))
-        .value();
+    const samples = _.map(
```

Comment:
Me too but I was changing underscore code to lodash code and did not know the correct way to use chaining.